pkg/client/applyconfiguration/monitoring/v1/scrapeclass.go

```diff
@@ -53,3 +58,16 @@ func (b *ScrapeClassApplyConfiguration) WithTLSConfig(value *TLSConfigApplyConfi
 	b.TLSConfig = value
 	return b
 }
+
+// WithRelabelings adds the given value to the Relabelings field in the declarative configuration
+// and returns the receiver, so that objects can be build by chaining "With" function invocations.
```

Comment:
I'm trying to have this `can be build` typo fixed in kubernetes upstream but let's see https://github.com/kubernetes/kubernetes/pull/123844 :p